### ISSUE TITLE
Some questions about the implementations of DGAdaIN

### ISSUE THREAD

**Issue by patrolli** [lovelyqian/AMeFu-Net#8]
Hello! I recently read the paper and the code, and I am a bit confusing about the implementation of DGAdaIN ( in `model_DGAdaIn.py`). 

According to the original paper, instance normalization is done along the temporal dimension (termed as L) of RGB features. However, in your codes, instance normalization is done along the last dimention of RGB features. I check the [document](https://pytorch.org/docs/stable/generated/torch.nn.InstanceNorm1d.html) that says `nn.InstanceNorm1d` will normalize along the last dimenstion of the input tensor. 

I am not sure which one (normalization along the temporal dimenstion or along the activations) is better and how they will affect the final results. Hoping for your help~

**Comment by lovelyqian** [comment]
Hi, thanks for your question.

:( sorry for noticing the issue now.

We apply the instance normalization along with the activations.  I guess what confused you is the concepts of temporal dimension (D) and the activation dimension (L). Specifically, given a video of BxDxL e.g., 6x16x2048,  we use the D to denote the number of frames, the L denotes the dimension of the activations.

**Comment by taxuewuhenxiaoer** [comment]
I'm confused of the "activation dimension (L)". Normally, it is the channel dimension. I'm very curious about the activation dimension (L).
